modules/heat_conduction/test/tests/semiconductor_linear_conductivity/steinhart-hart_linear.i

```diff
@@ -0,0 +1,92 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  xmax = 1.0
+  ymax = 1.0
+[]
+
+[Variables]
+  [./T]
+  [../]
+[]
+
+[AuxVariables]
+  [./elec_conduct]
+      order = FIRST
+      family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = MatDiffusion
```

Comment:
You'll either need to move this test into the combined module, or use the HeatConduction kernel. I think I'd recommend just using the HeatConduction kernel, even though it's ugly.